builders/build/historical-updates.md

```diff
@@ -854,3 +854,31 @@ This migration was executed at the following runtimes and blocks:
 | Moonbase Alpha |      RT1502      |    2112058    |
 
 For more information, you can review the [relative Nimbus PR](https://github.com/moonbeam-foundation/nimbus/pull/45){target=\_blank} and [Moonbeam PR on GitHub](https://github.com/moonbeam-foundation/moonbeam/pull/1400){target=\_blank}.
+
+### Foreign Assets Migration {: #foreign-assets-migration }
```

Comment:
H3's for migrations should list the pallet that the migration was run on and H4's should be the migration

Reply:
Wdyt about adding this section under XCM-Related Pallets?